django/publicmapping/publicmapping/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+REDIS_URL = 'redis://:{password}@{host}:{port}/{db}'.format(
```

Comment:
Why does this live here instead of in settings (or the templated settings extension)?

Reply:
This is really just to avoid doing the string formatting twice (this is used in `settings.py` and `celery.py`) :man_shrugging: 

I couldn't think of a great place to put it so I just threw it in here.

Reply:
I broke it out into a separate module (not in `settings.py`) because I running into an issue with `settings.py` being a celery dependency and the order in which settings were being built out. Something like celery being brought up before django and then django not having the extended settings yet. It also doesn't _need_ to be in settings. It's not really a setting, just a formatted string.

Reply:
Oh, yeah, docker-compose doesn't wait for dependencies to be "ready" before bringing them up, so what's happening is that `django` and `celery` are launched immediately after `config` is, before `config` has had a chance to write out the config file. So it might be easier just to duplicate this, if it's the only thing that `celery.py` needs, then `celery` might not have to depend on `django` at all?

Reply:
To be honest, I'm not entirely sure I understand what the dependencies between Celery and Django are.

In `celery.py` we refer to Django settings as per the docs' recommendation. Django uses the defined Celery app to queue up tasks. The containers themselves don't actually depend on each other at all. I think they just share code but as long as a Celery worker is running by the time a task is called, things will work. Maybe `django` should depend on `celery`?